### DISCUSSION TITLE
Backend fetch to nominatim.openstretemap.org failed

### DISCUSSION THREAD

**Discussion by philippstakenborg** [osm-search/Nominatim#3505] [Q&A]
Hi there,

I am not sure where to ask this question, and hope this is the right place. 

I have a plugin that uses the nominatim API to check for adresses. 
I use this plugin on one of my wordpress sites to check around 600 adresses after importing them. 
When using this plugin I constantly get a 503 error which looks like this:

> Error 503 Backend fetch failed
> 
> Backend fetch failed
> 
> Guru Meditation:
> 
> XID: 991552
> 
> Varnish cache server

I was trying to find out why this is happening and read more about the nominatim usage policy of 1 request per second. 
Could it be, that the plugin I use does not respect the usage policy and makes too many requests, so that I get said 503 error? 

What do you folks think? 

Thanks for your help
Philipp

**Comment by mtmail** [comment]
The nominatim.openstreetmap.org servers block IP addresses when too many requests come in. Your backend, maybe the Varnish server, then issues the HTTP 503 error.

Likely the Wordpress plugin sends the requests too fast. Is it a popular plugin, does the author offer support? Point them to https://operations.osmfoundation.org/policies/nominatim/

Also possible that in your setup all requests are routed through the Varnish cache server. In that case the nominatim.openstreetmap.org servers might see Varnish requests.
